### PR TITLE
Arm64 images

### LINKED ISSUE #1407: ARM docker image [feature request]
[Project board link](https://github.com/orgs/k8ssandra/projects/8/views/1?pane=issue&itemId=38624456)

Currently, [`thelastpickle/cassandra-reaper:3.3.3`](https://hub.docker.com/layers/thelastpickle/cassandra-reaper/3.3.3/images/sha256-edf98052e003820d8b66d0fa83f31693fd864d559f431de9b44b30a72b36d096?context=explore) is built only for `linux/amd64` architecture.

image

The[ Cassandra image ](https://hub.docker.com/layers/library/cassandra/4.1.3/images/sha256-3c272d54ff753fd4017256fd9763f0f81dc1702c61b381b8d819b288b7e4a083?context=explore) supports **ARM** architecture.
image

I would like to use `ARM` images** to run `cassandra-reaper` as a sidecar with `Cassandra` on `linux/arm64` machines like AWS Graviton (m7g, t4g, etc) in Kubernetes.

This will cut some costs and energy consumption. And might save the Planet from global warming ;)
#ARM #CostSaving

I also will be happier to run my [Thingsboard](https://thingsboard.io) deployment with Cassandra + Reaper completely on ARM.
image

Looking forward for your updates on ARM build.

Thank you!

### PR DESCRIPTION
**Pull request by adejanovski**
Fixes #1407 

Multi-arch Images built with this PR can be seen on [Docker Hub](https://hub.docker.com/r/thelastpickle/cassandra-reaper/tags).

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -364,15 +364,15 @@ jobs:
         run: |
           pip install pyyaml
           pip install ccm
-
+
```
Comment (emerkle826) [inline .github/workflows/ci.yaml]: nit:
would be nice to not have these single space character additions as changes.

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -457,7 +457,7 @@ jobs:
       - name: Compute version number
         run: |
           VERSION=$(printf 'VER\t${project.version}' | mvn help:evaluate | grep '^VER' | cut -f2)
-          if [ "${GITHUB_REF}" = "refs/heads/master" ]
+          if [ "${GITHUB_REF}" = "refs/heads/arm64-images" ]
```
Comment (emerkle826) [inline .github/workflows/ci.yaml]: question:
Shouldn't this be reverted to `master` once this PR is merged?

**Review by emerkle826 — APPROVED** [review]
Just 1 nit and one question. Otherwise this is good.

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (adejanovski) [reply .github/workflows/ci.yaml]: good catch, let me change it before merging

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (adejanovski) [reply .github/workflows/ci.yaml]: definitely, sorry about that

**Pull request merged**